### PR TITLE
[rustpotterks] Upgrade to version 2

### PR DESCRIPTION
**Pull request by GiviMAD**
Use new library version, which seems to perform better.

Note: Keyword files from the previous version are not compatibles.

### CODE REVIEW

File: bundles/org.openhab.voice.rustpotterks/README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ This voice service allows you to use the open source library Rustpotter as your
 
 Rustpotter provides personal on-device wake word detection. You need to generate a model for your keyword using audio samples.
 
+You can test library in your browser, using these web pages:
```
Comment (jlaur) [inline bundles/org.openhab.voice.rustpotterks/README.md:8]:
```suggestion
You can test library in your browser using these web pages:
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Only minor comments from my side.

File: bundles/org.openhab.voice.rustpotterks/README.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ This voice service allows you to use the open source library Rustpotter as your
...
+
+* [The spot demo](https://givimad.github.io/rustpotter-worklet-demo/), which include some example wakewords (but it's recommended to use your own).
+* [The model creation demo](https://givimad.github.io/rustpotter-create-model-demo/), it allows you to record compatible wav files and generate a wakeword file that you can test on the previous page.
```
[the remark this answers is not in the document]
Comment (GiviMAD) [reply bundles/org.openhab.voice.rustpotterks/README.md:11]: Thank for the review @jlaur, I'm working on the change. 

Would you be able to explain to me this review? I'm using the '*' character for other lists just after this one. Should I change the others? Or there is a difference between them?

[the remark this answers is not in the document]
Comment (jlaur) [reply bundles/org.openhab.voice.rustpotterks/README.md]: Please see #14371 and #13858. I'm not sure about exact reason, but probably it's a matter of consistency. Yes, if you can fix all, that would be great - I believe I just commented changed lines.

[the remark this answers is not in the document]
Comment (GiviMAD) [reply bundles/org.openhab.voice.rustpotterks/README.md]: Thanks for the references! I didn't realize that was one of the md improvements. On it.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**